app/src/main/java/com/example/dogbook/main/adapters/PostsAdapter.java

```diff
@@ -19,7 +19,7 @@
 
 public class PostsAdapter extends RecyclerView.Adapter<PostViewHolder> {
 
-    private List<Post> posts;
+    public static List<Post> posts;
```

Comment:
Why is this static now?  I think we would want this to tie to the actual instance of the adapter

Reply:
👍🏽, resolved in PR #54